playbook/app/pb_kits/playbook/pb_card/_card.tsx

```diff
@@ -1,42 +1,43 @@
-/* @flow */
+/* eslint-disable react/no-multi-comp */
 
 import React from 'react'
 import { get } from 'lodash'
 import classnames from 'classnames'
 
 import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
-import { globalProps } from '../utilities/globalProps'
+import { GlobalProps, globalProps } from '../utilities/globalProps'
 import { ProductColors, CategoryColors, BackgroundColors } from '../types'
```

Comment:
From initial investigation it looks like you just need to add `type` here.

```suggestion
import type { ProductColors, CategoryColors, BackgroundColors } from '../types'
```

Reply:
You are correct good sir. Typing that up now @Ivancandido11